extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -27,9 +36,33 @@ export default class MainController implements vscode.Disposable {
 		return Promise.resolve(true);
```

Comment:
Whats going on here? initialize databases returns a promise, but you don't awaiting it here.

Reply:
Copied the model from another extension.  As best I can tell, the intention was to allow the extension to initialize asynchronously, while being able to wait on that async initialization from any functional call in.  In any case, it's an obsolete model with `Deferred` removed.